src/Bitmap_cubical_complex/include/gudhi/Bitmap_cubical_complex_base.h

```diff
@@ -109,6 +109,15 @@ class Bitmap_cubical_complex_base {
    **/
   virtual inline std::vector<std::size_t> get_coboundary_of_a_cell(std::size_t cell) const;
 
+  /**
+   * This function finds a top-dimensional cell that is incident to the input cell and has 
+   * the same filtration value. In case several cells are suitable, an arbitrary one is 
+   * returned. Note that the input parameter is not necessarily a cube, it might also
+   * be an edge or vertex of a cube. On the other hand, the output is always indicating the position of
+   * a cube in the data structure. 
```

Comment:
If you read https://gudhi.inria.fr/doc/latest/group__cubical__complex.html , all the cells are called cube, a vertex is a cube of dimension 0, an edge is a cube of dimension 1, etc. Could you reformulate the last 2 sentences so they match the terminology?